From b5e63458036bd106adb55aa92c18bc35af7c57da Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Mon, 13 Jun 2016 16:42:35 +0800
Subject: [PATCH] ARM64: dts: rk3399: android: enable early console only

Disable the normal console of 8250, because we use fiq_debugger
console named ttyFIQ0.

Change-Id: I107cf950d646f8d42d2f3e7a6846762a59f3eb97
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index 71b29ba60a75..bd2b9ef46772 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -46,7 +46,7 @@
 	compatible = "rockchip,android", "rockchip,rk3399";
 
 	chosen {
-		bootargs = "console=uart,mmio32,0xff1a0000";
+		bootargs = "earlycon=uart8250,mmio32,0xff1a0000";
 	};
 
 	ramoops_mem: ramoops_mem {
-- 
2.35.3

